chainer/functions/connection/n_step_lstm.py

```diff
@@ -403,6 +403,13 @@ def n_step_lstm_base(
             'Use chainer.using_config')
         argument.assert_kwargs_empty(kwargs)
 
+    # Check input size consistency with xs and ws here.
+    x_in = xs[0].shape[1]
+    w_in = ws[0][0].shape[1]
+    if x_in != w_in:
+        raise ValueError('Inconsistent input size in input values and weight '
+                         'parameters')
```

Comment:
How about including the actual values to make the message a bit more friendly?

Reply:
Fixed!

Reply:
Thanks for the quick fix!